genesyscloud/tfexporter_state/tfexporter_state.go

```diff
@@ -0,0 +1,32 @@
+package tfexporter_state
+
+import (
+	"log"
+	"os"
+	"sync"
+)
+
+/*
+Export state is used to indicate whether an export is being done.  If the export state is set to true, then this should be
+a signal that any resources being exported should be reading their data from each resource's internal cache rather then the API.
+*/
+var exportState bool
+var once sync.Once
+
+// ActivateExporterState will be used to indicate that caching should be used to process requests.
+// We are setting this as an environment variable so we can experiment with it, without creating an attribute
+// on the resource
+func ActivateExporterState() {
+	cxCacheActive := os.Getenv("ENABLE_CX_CACHE")
```

Comment:
We need to refactor this.  We should not have an environment variable here as I put here only to be able turn caching on or if.  Make sure you remove the if cxCacheActive check and just leave the once.Do call

Reply:
removed the env check